pricewars_merchant.py

```diff
@@ -17,7 +17,11 @@ class PricewarsMerchant(metaclass=ABCMeta):
     # Save/Read token file in merchant directory
     TOKEN_FILE = Path(__file__).parent / 'auth_tokens.json'
 
-    def __init__(self, port: int, token: Optional[str], marketplace_url: str, producer_url: str, merchant_name: str):
+    def __init__(self, port: int, token: Optional[str], marketplace_url: str, producer_url: str, merchant_name: str, color: Optional[str]):
+        # default colors which are assigned to a merchant without it's own color
```

Comment:
its

Reply:
For multiple sentences, end a sentence with a period.

Reply:
Done.